global/meta/models.yml

```diff
@@ -732,6 +743,11 @@ meeting:
   imported_at:
     type: timestamp
     restriction_mode: B
+  language:
+    type: string
+    restriction_mode: A
+    description: Filled at meeting creation.
```

Comment:
you can remove the description, almost all fields are filled at meeting creation

Reply:
Removed. Done